sushichef.py

```diff
@@ -40,21 +55,44 @@ def construct_channel(self, **kwargs):
             book_info['absolute_url'] = page_url + book_info['relative_url']
 
             # Add book to channel tree
-            topic_node_source_id = 'topic/' + book_info['subject']
-            page_topic_node = TopicNode(title=book_info['subject'], source_id=topic_node_source_id)
-            channel.add_child(page_topic_node)
-            doc_node = DocumentNode(
-                title=book_info['title'],
-                description=book_info['description'],
-                source_id=book_info['source_id'],
-                license=get_license('CC BY-SA', copyright_holder='Stack Overflow'),
-                language='en',
-                files=[DocumentFile(path=book_info['absolute_url'], language='en')],
-            )
-            page_topic_node.add_child(doc_node)
+            book_node_source_id = 'topic/' + book_info['subject']
+            book_node = TopicNode(title=book_info['subject'], source_id=book_node_source_id)
+            channel.add_child(book_node)
+
+            # Use separate download directory for each book's pdf chunks. Avoids name conflicts between books
+            download_dir = 'downloads/book_' + str(book_counter).rjust(2, '0') + '--' + book_info['subject']
```

Comment:
Minor nitpick, but worth mentioning that this should be:

            download_dir = os.path.join('downloads', 'book_' + str(book_counter).rjust(2, '0') + '--' + book_info['subject'])

to make sure it works for Windows and Mac/Linux path separators.

Reply:
Ah, yes, thanks for pointing that out